docs/blockchain/contracts/nft-contract/nft-actions/mknftofr.a.experimental.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'mknftofr.a'
+order: 37
+
+---
+
+# mknftofr.a
+
+Make an offer on a given token.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the NFT factory from which the NFT was issued.
```

Comment:
Either "NFT" or "token" should be consistently used for all relevant action pages.

Reply:
update all to `uniq`